model/import/CsvItemImportHandler.php

```diff
@@ -82,6 +95,11 @@ public function import(
                 }
                 unset($itemImportReport);
             } catch (InvalidMetadataException $exception) {
+                $logger->error(sprintf(
```

Comment:
The same as here https://github.com/oat-sa/extension-tao-itemqti/pull/1760/files#r644031797